### PR TITLE
New todos!

### PR DESCRIPTION
**Pull request by iWatger**
Apologies for the delay. The aggregated assignments view now just shows unfinished assignments from your favorited courses

### CODE REVIEW

File: CanvasPlusPlayground/Views/AggregatedAssignmentsViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import SwiftUI
+
+struct AggregatedAssignmentsListCell: View {
+    let assignment: Assignment
+    let course: Course
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(assignment.name ?? "Missing name")
+            HStack {
```
Comment (rahulon12) [inline CanvasPlusPlayground/Views/AggregatedAssignmentsViewCell.swift:17]: Do we need an `HStack`?

File: CanvasPlusPlayground/Views/AggregatedAssignmentsView.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+//  CanvasPlusPlayground
+//
+//  Created by Ethan Fox on 10/18/24.
+//
+
+import SwiftUI
+
+struct AggregatedAssignmentsView: View {
+    @Environment(CourseManager.self) var courseManager
+    @State var course_assignments: [(Assignment, Course)] = []
```
Comment (rahulon12) [inline CanvasPlusPlayground/Views/AggregatedAssignmentsView.swift:12]: nit: naming convention
```suggestion
    @State var courseAssignments: [(Assignment, Course)] = []
```

**Review by rahulon12 — APPROVED** [review]
Couple nits; please upload a screenshot!

**Comment by iWatger** [comment]
resolved and merging!

**Pull request merged**